plugin-server/src/worker/vm/lazy.ts

```diff
@@ -249,9 +249,9 @@ export class LazyPluginVM {
     }
 
     private async processFatalVmSetupError(error: Error, isSystemError: boolean): Promise<void> {
-        this.hub.statsd?.increment('plugin_disabled_by_system', {
-            team_id: this.pluginConfig.team_id.toString(),
-            plugin_id: this.pluginConfig.plugin_id.toString(),
+        this.hub.statsd?.increment('plugin.disabled.by_system', {
+            teamId: this.pluginConfig.team_id.toString(),
+            plugin: this.pluginConfig.plugin?.name.toString() ?? '?',
```

Comment:
```suggestion
            plugin: this.pluginConfig.plugin?.name ?? '?',
```

Otherwise you're never hitting the ?? case. name should already be a string

Reply:
👍 Copypasta gone wrong